utils/roadie-backstage-entity-validator/src/validator.test.js

```diff
@@ -16,5 +16,9 @@ describe('validator',  () => {
   it('Should fail to validate with incorrect catalog-info that has an empty label', async () => {
     await expect(validator.validateFromFile('./sample/catalog-info-with-empty-label.yml')).rejects.toThrow("Error: Placeholder with name 'definition' is empty. Please remove it or populate it.");
   });
+
+  it('Should fail to validate with bad techdocs path', async () => {
```

Comment:
Should we also have a test where the tech mkdocs file is badly formatted?

Reply:
Sure I can try that out

Reply:
Discussed offline. We will encapsulate this in another ticket